compat/vcbuild/vcpkg_install.bat

```diff
@@ -34,6 +34,12 @@ REM ================================================================
 	@FOR /F "delims=" %%D IN ("%~dp0") DO @SET cwd=%%~fD
 	cd %cwd%
 
+	SET arch=x64-windows
+
+REM is vcpkg package installed and complied \\compat\\vcbuild\\vcpkg\\installed\\\$(VCPKGArch)\\include\\openssl\\ssl.h
+	dir vcpkg\installed\%arch%\include\openssl\ssl.h >nul 2>nul && GOTO :check_for_updates
```

Comment:
Oy. Should this not be `IF EXIST "..." GOTO ...`?

Relying on the exit status of `dir` when a file does, or does not exist, strikes me as odd in this context.

Reply:
Oy, oy, oy. I just realized that you copy-edited code already in the script... Tsk, tsk. Still, there is no good reason to copy flawed code, let's use `IF EXIST` instead.

Reply:
I'll try the `IF EXIST` approach. I never like `success=0` with the wrong/poor `&&` logic.